### PR TITLE
Ground/excited states of electron layers are implemented.

### PR DESCRIPTION
**Pull request by artemkondyukov**
Plus Linter and stylish-haskell suggestions applied.

### CODE REVIEW

**Comment by artemkondyukov** [comment]
Hi, Krzysztof!
My PR won't compile because GHC <8 lacks <$> operator
I believe, it's reasonable to use more recent version of compiler

File: src/Radium/Element.hs [hunk 1]
```diff
@@ -195,64 +205,125 @@ configExceptions = Map.fromList [ (24, [2, 8, 13, 1])
...
--- > covalentBounds e == 2 
+-- > covalentBounds e == 2
 covalentBounds :: Element -> Int
 covalentBounds e = min n (8-n)
-    where n = valanceElectrons e
+  where n = valanceElectrons e
+
+
+-- Get list of all possible coordination numbers (valences)
+possibleValences :: Element -> [Int]
```
Comment (klangner) [inline src/Radium/Element.hs:243]: It would be useful to start comments with '-- |' for all exposed functions.
Comments which starts this way will show in the documentation. Comments which starts with '--' don't.

File: src/Radium/Element.hs [hunk 2]
```diff
@@ -195,64 +205,125 @@ configExceptions = Map.fromList [ (24, [2, 8, 13, 1])
...
+
 
 -- Fill shells with given number of electrons
 fillShells :: Int -> [( Int  -- Shell number
-                    , Int  -- Shell type
-                    , Int  -- #Electrons
-                      )]
+                      , Int  -- Shell type
+                      , Int  -- #Electrons
+                        )]
```
Comment (klangner) [inline src/Radium/Element.hs:322]: Just as a note. 
Do all triples have the same meaning like in this function?
I wonder if it is better to provide ADT for this type instead of generic Tuple (Triple).

But even if yes then it is better so leave it for another PR.

**Review by klangner — APPROVED** [review]
Hi Artem,

Thank you for this PR.
I will merge and fix travis build. 

But I think 2 things are worth considering later:
  * Changing the comment style for public functions
  * Replacing Triple with ShellConfig data structure.

**Comment by klangner** [comment]
Thank you :-)

File: src/Radium/Element.hs (hunk 1 above)
Comment (artemkondyukov) [reply src/Radium/Element.hs:243]: Yeah, I'll fix that in my later contributions.

File: src/Radium/Element.hs (hunk 2 above)
Comment (artemkondyukov) [reply src/Radium/Element.hs:322]: Sure, we need ADT here, probably it's better to open an issue?

Comment (klangner) [reply src/Radium/Element.hs:322]: Yes. Good idea. 
Do you want to do it?Or should I do it?

BTW there is one more thing. Since you added a lot of code to the Element.hs. I think it would be good to add your name to the copyright in the header. Or instead of copyright line add list of authors with your name.

Of course if it is ok with you?

Comment (artemkondyukov) [reply src/Radium/Element.hs:322]: Please, open the issue and inform me in case you start to implement it, otherwise I'll do this in a couple of days.
I'll add my name several PRs later, if you don't mind :)
